packages/components/src/drop-zone/provider.js

```diff
@@ -109,10 +109,19 @@ class DropZoneProvider extends Component {
 		}
 
 		this.setState( {
-			isDraggingOverDocument: false,
 			hoveredDropZone: -1,
+			isDraggingOverDocument: false,
+			isDraggingOverElement: false,
 			position: null,
+			type: null,
 		} );
+
+		this.dropZones.forEach( ( dropZone ) => dropZone.setState( {
```

Comment:
This change makes this function different from the previous behavior right? I wonder if it has an impact.

Reply:
Previously, we were not calling `updateState` on all dropzones in this method. Why is that different?

Reply:
mmm, how so? In [master](https://github.com/WordPress/gutenberg/blob/master/packages/components/src/drop-zone/provider.js#L64), `resetDragState` calls `updateState` for all drop zones. It doesn't keep a registry of which dropzones were updated so it needs to call them all when the drop has finished or been aborted.

Reply:
mm you're right nevermind, I was looking at a single commit and the updateState had been removed on a previous one.